first_experiment.tex

```diff
@@ -0,0 +1,68 @@
+\section{Diversity analysis}
+
```

Comment:
There should be an introduction before this stating *why* we do this diversity analysis. I guess that's on us.